### PR TITLE
Bugfix/fix font selection issue

### PR DESCRIPTION
**Pull request by KarthikeyaGit**
This PR fixes a problem where I couldn't choose fonts because of a loop. Now, instead of putting getFonts() directly in the HTML, I saved the font details in a variable named fonts. This should make picking fonts easier without any issues.

### CODE REVIEW

**Comment by wfpena** [comment]
This is awesome @KarthikeyaGit . Taking a look at the PR.

One thing, could you remove the package.json ?

**Comment by wfpena** [comment]
Also, the changes on yarn.lock, seems like you used npm. I plan on changing to npm in the future but I think we can just push the bugfix in this case.

**Comment by KarthikeyaGit** [comment]
> Also, the changes on yarn.lock, seems like you used npm. I plan on changing to npm in the future but I think we can just push the bugfix in this case.

Done.

File: .gitignore [hunk 1]
```diff
@@ -99,3 +99,9 @@ testem.log
 # System Files
 .DS_Store
 Thumbs.db
+
+# Ignore package lock files generated by npm and Yarn
```
Comment (wfpena) [inline .gitignore:103]: no need to ignore those files here, just need to remove the changes from the commit

File: yarn.lock [hunk 2]
```diff
@@ -1,8979 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (wfpena) [inline yarn.lock:1]: we should not delete this file from the repo, it should be here but change when we change a dependency.

**Review by wfpena — CHANGES_REQUESTED** [review]

**Comment by wfpena** [comment]
@KarthikeyaGit added a review with some comments

File: .gitignore (hunk 1 above)
Comment (KarthikeyaGit) [reply .gitignore:103]: ok

**Comment by wfpena** [comment]
@KarthikeyaGit  Closing this PR in favor of a refactor on version 1.1.6. https://github.com/wfpena/angular-wysiwyg/pull/31

**Pull request closed without merging**